### ISSUE TITLE
Is stackedit is being maintained?

### ISSUE THREAD

**Issue by lefuturiste** [benweet/stackedit#1641]

**Comment by iaon** [comment]
Yeah. It's insane. #1 google search result for "online markdown editor" 6 years of development and then... nothing.

**Comment by nzewail** [comment]
I'm wondering the same thing. I like the editor but don't really want to be using something that isn't being maintained

**Comment by katomaso** [comment]
@benweet what about moving the project under some OSS organization and giving maintainer rights to multiple people? Would that work? I like this project too would love to see it being developed further.

**Comment by jsuar** [comment]
@katomaso Given the license couldn't folks who are interested fork the project into a new org and begin actively maintaining it there?

**Comment by ortonomy** [comment]
@katomaso @jsuar -- I'm down for forking to a new repo and hosting in my org. Is there really interest in this?
Might have to disable a bunch of features (pay walls, server side PDF generation, but we could get it going...)

**Comment by jsuar** [comment]
@ortonomy I actually switched to https://github.com/marktext/marktext so I probably wouldn't participate. Thanks for the offer 👍

**Comment by ortonomy** [comment]
No worries @jsuar -- just checked out marktext, it's lovely but currently borked on macos and you can't sync to a cloud store, it's all local files... less happy about that.

**Comment by jsuar** [comment]
@ortonomy I hear ya. It was a hell of a search to find a good live markdown editor. Marktext looks nice but does have some annoying bugs. I'm getting around the local files issue by using a Github repo which is less than ideal. I've been floating an idea of integrating some kind of encryption either before pushing to the repo or wrapped around the entire editing process (so I'd have encryption at rest too). I really want to "own" as much of the data so I avoided a lot of the other editors out there. Using a repo or cloud but encrypting before uploading would be pretty sweet. Kind of the local first mindset.

**Comment by bendtherules** [comment]
@ortonomy Interested, if you make a fork. First check if there is a good existing fork.

**Comment by cburkins** [comment]
I like the looks of StackEdit, but with the github repo seeing no commits for 14 months, I'm wondering if it's a good choice ?   I've got my account setup, and seems to be syncing with Google Drive just fine, so going to try it out.

**Comment by MarkGStacey** [comment]
> @katomaso @jsuar -- I'm down for forking to a new repo and hosting in my org. Is there really interest in this?
> Might have to disable a bunch of features (pay walls, server side PDF generation, but we could get it going...)

Yeah I could be interested

**Comment by ortonomy** [comment]
@bendtherules -- good point. But there are 2.4K Forks! what do we do!?

**Comment by thwaller** [comment]
I had this same question, leading me to this issue 1641.  Unfortunately, it was after I started using this that I discovered there is no maintenance at all. I would for sure have interest in a fork of this that would continue its maintenance, as well as the paywall features. If the developer is not active on the project, there should be no payments to use it. Given the age of this issue, I will start looking for another option, especially one that I can share, even if it is restricted to those in a GSuite org.

**Comment by abartov** [comment]
I too am interested in a maintained alternative. If anyone found one since 2020, I'd love to get a pointer!

**Comment by KaKi87** [comment]
I used [Useful Forks](https://useful-forks.github.io) to try and find one, without success. Lots of rebrands, minor edits, infra adjustments, etc., but nothing that end users could rely on.

There is [one](https://github.com/knackstedt/stackedit) that was really close, and it actually implemented a rare innovative Typora/MarkText-style WYSIWYG editor :

<img width="1271" height="647" alt="Image" src="https://github.com/user-attachments/assets/0d2a13de-6c8f-4a0e-a4f4-20fcd3e66a96" />

Unfortunately, it isn't exactly daily-driveable and it has been over a year since their maintainer's last commit, despite being active on other (mostly private) projects, unlike StackEdit's creator who completely abandoned GitHub though seem active on Linkedin for commercial purposes.

**Comment by abartov** [comment]
For the benefit of anyone wandering here with the same question, I finally settled on this maintained repo:
https://github.com/SamTV12345/stackedit-react

And have recently managed to get [a feature PR merged](https://github.com/SamTV12345/stackedit-react/pull/79).

**Comment by KaKi87** [comment]
Oh. It's a rewrite rather than a fork though, pretty much stealing the *StackEdit* name & logo, and lacking many settings.
